src/domain/entities/schemas/lenient-page.schema.factory.ts

```diff
@@ -1,21 +0,0 @@
-import { TypeOf, z } from 'zod';
-import { Page } from '@/domain/entities/page.entity';
-
-/**
- * Builds a lenient page schema that filters out invalid items from the results array.
- */
-export function buildLenientPageSchema<T extends z.ZodTypeAny>(
```

Comment:
I moved this to the factory file as the base schema was extracted to `BasePageSchema`.